crypto-volatility-index/adapter.js

```diff
@@ -0,0 +1,34 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+const cryptoVolatilityIndex = require('./src/cryptoVolatilityIndex.js')
+
+const bootstrap = () => {
+  console.log('Bootstrapping CVX calculation')
+  cryptoVolatilityIndex.scheduleCron()
+}
+
+const execute = (input, callback) => {
+  const validator = new Validator(input)
+  if (validator.error) return callback(validator.error.statusCode, validator.error)
+
+  const jobRunID = validator.validated.id
+  console.log('jobRunID:', jobRunID)
```

Comment:
This seems unnecessary to log. The internal logger should log the payload regardless.